projects/demo/src/index.html

```diff
@@ -16,14 +16,10 @@
     />
     <script
       defer
-      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDg4a_K_2lB2rvR3VbFch_rmQROF0oCDjE"
+      src="https://maps.googleapis.com/maps/api/js"
```

Comment:
So Google Maps works without an API key for development purposes? Good.

Reply:
Yes :=)